render/detailed/summary.go

```diff
@@ -29,6 +29,18 @@ const (
 	MarathonAppIDEnv             = "MARATHON_APP_ID"
 )
 
+var (
+	shapesByTopology = map[string]string{
+		report.Host:           Circle,
+		report.Process:        Square,
+		report.Pod:            Heptagon,
+		report.Service:        Heptagon,
+		report.Container:      Hexagon,
+		report.ContainerImage: Hexagon,
+		render.Pseudo:         Cloud,
+	}
+)
```

Comment:
I'm not a fan of hard coded things like this; perhaps we should add a `shape` field to the Topology struct?